components/dashboard/sections/HostDashboardHostedCollectives.tsx

```diff
@@ -55,6 +58,19 @@ const filters: FilterComponentConfigs<z.infer<typeof schema>> = {
     ),
     valueRenderer: ({ value, intl }) => formatHostFeeStructure(intl, value),
   },
+  type: {
+    labelMsg: defineMessage({ id: 'Type', defaultMessage: 'Type' }),
+    Component: ({ intl, ...props }) => (
+      <ComboSelectFilter
+        options={Object.values(HostedCollectiveTypes).map(value => ({
```

Comment:
If we want to allow multi selection:
```suggestion
        isMulti
        options={Object.values(HostedCollectiveTypes).map(value => ({
```